eyewear_shop/data/product_public_category.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="product_public_category_1" model="product.public.category">
+        <field name="name">Sunglasses</field>
+    </record>
+    <record id="product_public_category_10" model="product.public.category">
+        <field name="name">Others</field>
+    </record>
+    <record id="product_public_category_11" model="product.public.category">
+        <field name="name">Storage</field>
+        <field name="parent_id" ref="product_public_category_10"/>
+    </record>
+    <record id="product_public_category_2" model="product.public.category">
+        <field name="name">Carrera</field>
+        <field name="parent_id" ref="product_public_category_1"/>
+    </record>
+    <record id="product_public_category_3" model="product.public.category">
+        <field name="name">Maui Jim</field>
+        <field name="parent_id" ref="product_public_category_1"/>
+    </record>
+    <record id="product_public_category_4" model="product.public.category">
+        <field name="name">Oakley</field>
+        <field name="parent_id" ref="product_public_category_1"/>
+    </record>
+    <record id="product_public_category_5" model="product.public.category">
+        <field name="name">Ray-ban</field>
+        <field name="parent_id" ref="product_public_category_1"/>
+    </record>
+    <record id="product_public_category_6" model="product.public.category">
+        <field name="name">Contact Lenses</field>
+    </record>
+    <record id="product_public_category_7" model="product.public.category">
+        <field name="name">Johnson&amp;Johnson</field>
+        <field name="parent_id" ref="product_public_category_6"/>
+    </record>
+    <record id="product_public_category_8" model="product.public.category">
+        <field name="name">Bausch + Lomb</field>
+        <field name="parent_id" ref="product_public_category_6"/>
+    </record>
+    <record id="product_public_category_9" model="product.public.category">
+        <field name="name">Dolce &amp; Gabbana</field>
+        <field name="parent_id" ref="product_public_category_1"/>
+    </record>
```

Comment:
Following the previous remark and the discussion on the task, I guess we can remove brands from public categories as well.

Reply:
Yes, obviously :+1: